cmd/store/keys.go

```diff
@@ -301,11 +302,15 @@ func Match(pubkey string, kp nkeys.KeyPair) bool {
 }
 
 func keyFromFile(path string) (nkeys.KeyPair, error) {
-	path, err := filepath.Abs(path)
+	hfp, err := homedir.Expand(path)
```

Comment:
I thought Abs would expand relative but maybe not? What is homedir? Is this another dependency?

Reply:
We already had the dependency - yes - abs doesn't resolve ~.